packages/core/src/routes/authn.ts

```diff
@@ -144,4 +150,73 @@ export default function authnRoutes<T extends AnonymousRouter>(
       return next();
     }
   );
+
+  /**
+   * SAML SSO connector's assertion consumer service endpoint
+   *
+   * @param connectorId The connector id.
+   * @property body The SAML assertion response body.
+   * @property body.RelayState We use this to find the connector session.
+   * @returns Redirect to the redirect uri find in the connector session storage.
+   *
+   * @remark
+   * This API is used to handle SSO SAML assertion from the identity provider.
+   * Validate and parse the SAML assertion, then store the assertion data to the connector session storage.
+   * Redirect to the redirect uri find in the connector session storage.
+   * The above endpoint will be deprecated in the future.
+   */
+  router.post(
+    `/authn/${ssoPath}/saml/:connectorId`,
+    koaGuard({
+      body: z.object({ RelayState: z.string() }).catchall(z.unknown()),
```

Comment:
double check that pascal case is defined by the standard. if so, better to specify in the comment/doc

Reply:
Yes it is defined by the standard.